README.windows.md

```diff
@@ -242,6 +242,14 @@ If you are building for 64-bit windows, the steps are essentially the same. Just
 
 - run gdb under the windows console (cmd) instead. gdb [does not function properly](https://www.cygwin.com/ml/cygwin/2009-02/msg00531.html) under mintty with non-cygwin applications.
```

Comment:
is it worth suggesting `cmd /c start` for popping out from `mintty` to `console`?

fwiw, i also find that the mingw gdb can be more accurate and useful than the cygwin gdb